src/components/Root/Root.tsx

```diff
@@ -83,7 +71,7 @@ const Root: FC = () => {
   return (
     <>
       {isReady && <AppRoutes />}
-      {isReady && <AccountModal />}
+      {isReady && hasIntegration && <AccountModal />}
```

Comment:
What's the value of adding this check?

Reply:
Not to render AccountModal component at all in case no integration exists in the configuration file.

Reply:
I guess that makes sense as a small optimization. The modal won't ever show anything in this case, but there are some side effects that will get rerendered.

Reply:
You can show it when manually inserting query params even for configs with no integration part :(
But I can also use this check inside `AccountModal` component.

Reply:
Oh, OK. This is good then! Thanks for explaining!